data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,17 @@
+/* exported countValues */
+
+function countValues(stack) {
+  let count = 0;
+  let topValue;
+  topValue = stack.peek();
+  if (topValue === undefined) {
+    return count;
+  } else {
```

Comment:
You don't really need this extra check, your while loop would just never run if `topValue` is `undefined`